src/patternfly/components/Table/docs/table-checkboxes-actions.md

```diff
@@ -12,18 +12,16 @@ Note: When header cells are empty or they contain interactive elements, `<th>` s
 ### Thead Accessibility
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `aria-label="[descriptive text]"` | `.pf-c-table__check` | Provides an accessible name for checkbox header cells. **Required** |
+| `aria-label="[descriptive text]"` | `.pf-c-table__check input[type="checkbox"]` | Provides an accessible name for the checkbox. **Required** |
 
 ### Tbody Accessibility
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `aria-labelledby="{title_cell_id} {element_id}"` | `.pf-c-check__input`, `.pf-c-button`  | Provides an accessible name for special cells containing interactive elements. **Required** |
-| `aria-label="[descriptive text]"`  | `.pf-c-check__input`, `.pf-c-button`  | Provides an accessible name for `checkboxes` and `buttons`. **Required** |
-| `id`  | title cell `<th>` > `<span>`, `.pf-c-check__input`, `.pf-c-button`  | Provides an accessible description for special cells. **Required** |
+| `aria-labelledby="{title_cell_id}"` | `.pf-c-table__check input[type="checkbox"]` | Provides an accessible name for the checkbox. **Required** |
+| `id`  | title cell `<th>` > `*` | Provides an accessible description for the checkbox. **Required** |
```

Comment:
In chatting with @TheRealJon about [this implementation for react](https://github.com/patternfly/patternfly-react/issues/1460), he had mentioned that setting up the `aria-labelledby`/`id` relationship is more complex than just using `aria-label`. I would still keep our examples as they are, but since the react component will have a different (but also valid) method for providing the accessible name, should we show that variation as well?

If so, I would just add "or `aria-label="[descriptive text]"` for line 20. For line 21, I'm not sure if it's better to remove that and assume that they would know that `{title_cell_id}` means they need to add an `id` somewhere, or if we should say "**Required if** using `aria-labelledby` for `.pf-c-table__check input[type="checkbox"]`

And this is a total nit, but should we use consistent formatting for `[descriptive text]` and `{title_cell_id}`?